modules/ROOT/pages/keyword-glossary.adoc

```diff
@@ -58,6 +58,14 @@ Typically used when modifying or importing large amounts of data.
 | Schema
 | Create a constraint that ensures all relationships with a particular type have all the specified properties and that the combination of property values is unique; i.e. ensures existence and uniqueness.
 
+| xref::constraints/syntax.adoc#constraints-syntax-create-node-prop-type[CREATE CONSTRAINT [property_type\] [IF NOT EXISTS\] FOR (n:Label) REQUIRE n.property IS :: type [OPTIONS {}\]]
+| Schema
+| Create a constraint that ensures that for all nodes with a particular label and property, the property have a certain property type (given by `type`).
```

Comment:
```suggestion
| Create a constraint ensuring that a property belonging to certain nodes has the required property type (given by `type`).
```

Reply:
This is also following existing patterns (just adding the type requirement) so if we want to update this we should consider if we want to update the other constraints as well